web/services/getPersonalSummaryByDateService.php

```diff
@@ -129,7 +129,27 @@
         }
         if ($minutes < 10)
             $minutes = "0" . $minutes;
-        $pendingHolidays = $hours . ":" . $minutes;
+
+        $currentJourney = 8;
+        $journeys = UsersFacade::GetUserJourneyHistoriesByIntervals($initDate, $date, $userVO->getId());
+        if(count($journeys)==1) {
+                $currentJourney = $journeys[0]->getJourney();
+        }
+
+        if ($hours > $currentJourney*7) {
```

Comment:
I'd use $currentJourney * 5 (that means 40 in an 8-hour journey).

Reply:
Indeed, 5 work days, not 7, that would mean a whole week.